### ISSUE TITLE
1.0 support

### ISSUE THREAD

**Issue by jroper** [jrudolph/sbt-cross-building#33]
We'll need to add sbt 1.0 support to `sbt-cross-plugin`, since `sbt-cross-plugin` is going to become very popular again since sbt 1.0 breaks binary compatibility with 0.13.

I've got a question though, is it worth cross building `sbt-cross-plugin` against 0.11 and 0.12?  Of course, it can still support cross building against these, but whether th `sbt-cross-plugin` itself is cross built against these, I don't see any value in maintaining that, I don't think any serious sbt plugin today that wanted the latest `sbt-cross-building` plugin would be using anything earlier than 0.13.5, I think we should make the minimum version for `sbt-cross-plugin` 0.13.5 (so it can use auto plugins), and then cross build it against 0.13 and 1.0.

**Comment by jrudolph** [comment]
> I've got a question though, is it worth cross building sbt-cross-plugin against 0.11 and 0.12?

Yes, I'll probably just drop any support for 0.11 and 0.12 versions. I don't think they are still worse the hassle.

**Comment by jrudolph** [comment]
I pushed a 0.9.0-M1 version with the changes from #35 to sonatype / maven central. This plugin itself is currently only built for 0.13.13, so you can only forward cross-compile your plugins.
